README.md

```diff
@@ -9,21 +9,20 @@ This is the driver for JDBC connectivity to a cluster running with OpenSearch SQ
 | ----- | ----------------------------------------------- |
 | Build | [![JDBC CI][jdbc-build-badge]][jdbc-build-link] |
 
-[jdbc-build-badge]: https://github.com/opensearch-project/sql/actions/workflows/sql-jdbc-test-and-build-workflow.yml/badge.svg
-[jdbc-build-link]: https://github.com/opensearch-project/sql/actions/workflows/sql-jdbc-test-and-build-workflow.yml
+[jdbc-build-badge]: https://github.com/opensearch-project/sql-jdbc/actions/workflows/sql-jdbc-test-and-build-workflow.yml/badge.svg
+[jdbc-build-link]: https://github.com/opensearch-project/sql-jdbc/actions/workflows/sql-jdbc-test-and-build-workflow.yml
 
 ## Specifications
 
 The driver is compatible with JDBC 4.2 specification and requires a minimum of Java 8.
 
 ## BI Tool Connectors
 
-* [Power BI Desktop](../bi-connectors/PowerBIConnector/README.md)
-* [Tableau Desktop](../bi-connectors/TableauConnector/README.md)
+* [Tableau Desktop](bi-connectors/TableauConnector/README.md)
 
 ## Download and Installation
 
-The driver is available for download from [Artifacts page](https://opensearch.org/artifacts) on OpenSearch.org at the very bottom and from [automated CI workflow](https://github.com/opensearch-project/sql/actions/workflows/sql-jdbc-test-and-build-workflow.yml).
+The driver is available for download from [Artifacts page](https://opensearch.org/artifacts) on OpenSearch.org at the very bottom and from [automated CI workflow](https://github.com/opensearch-project/sql-jdbc/actions/workflows/sql-jdbc-test-and-build-workflow.yml).
```

Comment:
can u plz also add few words about that now this maven artifact is deployed into nexus ?
here - https://repo1.maven.org/maven2/org/opensearch/driver/opensearch-sql-jdbc

Reply:
Fixed in 305e2a4.